test/test_distributions.py

```diff
@@ -806,6 +818,17 @@ def test_binomial_extreme_vals(self):
         self.assertAlmostEqual(bin1.log_prob(torch.tensor([float(total_count)]))[0], 0, places=3)
         self.assertEqual(float(bin1.log_prob(torch.tensor([float(total_count - 1)])).exp()), 0, allow_inf=True)
 
+    def test_binomial_vectorized_count(self):
+        set_rng_seed(0)
+        total_count = torch.tensor([[4., 7.], [3., 8.]])
+        bin0 = Binomial(total_count, torch.tensor(1.))
```

Comment:
With a `probs` this high, all you need is a single sample 😉

Reply:
Yup, this is doing an exact match! I am drawing way more samples for `bin1` with `p=0.5` to make sure that the invariance holds.